### DISCUSSION TITLE
do wo have a plan ,remove etcd , save data localstoage ? no use any other like mysql ,etcd ,zookeeper,keep simple?

### DISCUSSION THREAD

**Discussion by jcy1001** [apache/kvrocks-controller#151] [General]
do wo have a plan ,remove etcd , save data localstoage ? not use any other like mysql ,etcd ,zookeeper,keep simple?

**Comment by git-hulk** [comment]
Yes, we have proposed this as one of GSoC project, please refer: https://cwiki.apache.org/confluence/display/COMDEV/GSoC+2024+Ideas+list#GSoC2024Ideaslist-[GSoC][Kvrocks]SupportembeddedstorageforKvrocksclustercontroller
